proposals/akri-resources-status.md

```diff
@@ -0,0 +1,78 @@
+# Status of akri resources
+
+Currently, both akri managed resources (`Instances` and `Configurations`) only have a `spec` and `metadata` field, this doesn't allow the akri
+components to give feedback to the user. In order to do so, we can add a `status` field to the resources, this document describes how this field
+should be populated.
+
+## Instances
+
+For an instance, the status should give insight about the broker resources (if any), to do that, you can use the following conditions:
+
+- Healthy: The instance is currently detected by an agent, "True" (instance detected) and "Unknown" (in grace period) are the only valid statuses
+- BrokerScheduled: The broker resources are all created (absent if no broker)
```

Comment:
The Instance can have multiple slots, each slot can be either free, used as Instance level resource or used as Configuration level resource.  How does the status field reflects the different status of slots?  
For Configuration level resource, there won't be 'broker' associates with it, but the resource is allocated for a workload.  How does the status field reflects this usage?

Reply:
The goal of that specific condition is not to reflect the status of all slots, but just to tell if the controller managed to create all broker resources requested by the configuration.

Reply:
I see, sounds good.